3rdparty/flatbuffers/include/flatbuffers/flatc.h

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Do we need this file?

Reply:
If we do, then we can build those and we don't need the ansible stuff. Otherwise let's remove it?